traffic_portal/test/integration/PageObjects/SideNavigationPage.po.ts

```diff
@@ -54,6 +54,10 @@ export class SideNavigationPage extends BasePage{
     private lnkCoordinates = element(by.xpath("//a[@href='/#!/coordinates']"));
     private lnkRegions = element(by.xpath("//a[@href='/#!/regions']"));
     private lnkASNs = element(by.xpath("//a[@href='/#!/asns']"));
+    //Navigation for Jobs
+    private propTools = "//div[@id='sidebar-menu']//a[contains(text(),'Tools')]"
```

Comment:
CSS selectors are easier to read and understand than XPath, the equivalent of this would be `element.byCSSContainingText("#sidebar-menu a", "Tools"`)`